bin/install-aws-cli

```diff
@@ -1,32 +0,0 @@
-#!/bin/bash
```

Comment:
I also poked in there as well as the ASF Image monitor project (since ASF owned this for a time), but I never figured out what the ASF Image Monitor was monitoring.

Reply:
I also poked in there as well as the ASF Image monitor project (since ASF owned this for a time), but I never figured out what the ASF Image Monitor was monitoring.